services/dynamic-scheduler/tests/unit/test_services_catalog.py

```diff
@@ -0,0 +1,123 @@
+# pylint:disable=redefined-outer-name
+# pylint:disable=unused-argument
+
+
+import urllib.parse
+from collections.abc import Iterator
+
+import pytest
+import respx
+from fastapi import FastAPI
+from models_library.api_schemas_catalog.services_specifications import (
+    ServiceSpecifications,
+)
+from models_library.service_settings_labels import SimcoreServiceLabels
+from models_library.services import ServiceKey, ServiceVersion
+from models_library.users import UserID
+from pydantic import TypeAdapter
+from pytest_simcore.helpers.typing_env import EnvVarsDict
+from simcore_service_dynamic_scheduler.services.catalog import CatalogPublicClient
+
+
+@pytest.fixture
+def app_environment(
+    disable_redis_setup: None,
+    disable_rabbitmq_setup: None,
+    disable_service_tracker_setup: None,
+    disable_deferred_manager_setup: None,
+    disable_notifier_setup: None,
+    disable_status_monitor_setup: None,
+    app_environment: EnvVarsDict,
+) -> EnvVarsDict:
+    return app_environment
+
+
+@pytest.fixture
+def simcore_service_labels() -> SimcoreServiceLabels:
+    return TypeAdapter(SimcoreServiceLabels).validate_python(
+        SimcoreServiceLabels.model_json_schema()["examples"][1]
+    )
+
+
+@pytest.fixture
+def service_specifications() -> ServiceSpecifications:
+    return TypeAdapter(ServiceSpecifications).validate_python({})
+
+
+@pytest.fixture
```

Comment:
please reuse [`pytest_simcore.faker_users_data` fixtures](https://github.com/itisfoundation/osparc-simcore/blob/5732a12e07e63d5ce55010ede9b9ab543bb9b278/packages/pytest-simcore/src/pytest_simcore/faker_users_data.py)

we have repeatedly discussed about this :-) ... can you please change all the fixtures in this service to use those.

There are many advantages by reusing these fixtures.